apps/reply/lib.js

```diff
@@ -0,0 +1,75 @@
+exports.reply = function (options) {
+  var keyboard = "textinput";
+  try {
+    keyboard = require(keyboard);
+  } catch (e) {
+    keyboard = null;
+  }
+
+  function constructReply(msg, replyText, resolve) {
+    var responseMessage = {msg: replyText};
+    if (msg.id) {
+      responseMessage = { t: "notify", id: msg.id, n: "REPLY", msg: replyText };
+    }
+    E.showMenu();
+    layout.setUI();
+    layout.render();
+    if (options.sendReply == null || options.sendReply) {
+      Bluetooth.println(JSON.stringify(responseMessage));
+    }
+    resolve(responseMessage);
+  }
+
+  return new Promise((resolve, reject) => {
+    var menu = {
+      "": {
+        title: options.title || /*LANG*/ "Reply with:",
+        back: function () {
+          E.showMenu();
+          layout.setUI();
+          layout.render();

```

Comment:
Do we need to do these when the user's exiting?

Reply:
I had added this to fix a white screen during testing - just removed it as the calling app can redraw after the promise resolves!

Reply:
Cool, sounds good!